src/review-form.ts

```diff
@@ -160,6 +210,35 @@ export class ReviewForm extends LitElement {
     `;
   }
 
+  private async setupRecaptcha(): Promise<void> {
+    this.recaptchaWidget = await this.recaptchaManager?.getRecaptchaWidget();
+    this.recaptchaError = false;
+  }
+
+  private async handleSubmit(e: Event): Promise<void> {
+    e.preventDefault();
+    if (!this.reviewForm.reportValidity()) return;
+    if (!this.recaptchaWidget) {
+      this.recaptchaError = true;
+      return;
+    }
+
+    try {
+      const recaptchaToken = await this.recaptchaWidget?.execute();
```

Comment:
Since you checked for `this.recaptchaWidget` above, you should be able to get rid of the question mark here